config.js

```diff
@@ -18,6 +18,15 @@ const config = {
   FC_SCOPES: 'given_name family_name birthdate gender birthplace birthcountry email preferred_username address phone',
   DGFIP_SCOPES: 'dgfip_rfr dgfip_nbpart dgfip_sitfam dgfip_pac dgfip_aft',
   DGFIP_DATA_FD_PATH: '/situations/ir/assiettes/annrev/2018',
+  OPENID_ERRORS: {
```

Comment:
this is not real config as it is not platform dependant. I suggest you put this in another file 'locale.fr.js' for instance

Reply:
ok... out of scope... we have to it clean and fast. This (excellent) idea opens a lot of questions about architecture that are out of the ticket. Please discuss with Stephane or/and Guillaume.

Reply:
more complicated than you think but ok, done